docs/source/en/features/distributed_adafactor.md

```diff
@@ -0,0 +1,114 @@
+
+### API Reference
+
+{{ autodoc:colossalai.nn.optimizer.distributed_adafactor.DistributedAdaFactor }}
+
+## Hands-On Practice
+We now demonstrate how to start Distributed Adafactor with booster API.
+### step 1. Import libraries
+
+```python
+import torch
+from torch import nn
+import torch.distributed as dist
+from transformers import LlamaModel, LlamaConfig
+
+from colossalai.cluster import ProcessGroupMesh
+from colossalai.shardformer.layer import Linear1D_Col, Linear1D_Row
+from colossalai.nn.optimizer.distributed_adafactor import DistributedAdaFactor
+from colossal_llama2.dataset.loader import load_tokenized_dataset
+from colossalai.booster.plugin import GeminiPlugin, HybridParallelPlugin, LowLevelZeroPlugin
+```
+
+### step 2. Initialize Distributed Environment and Parallism Group
+We then need to initialize distributed environment. For demo purpose, we uses `colossalai.launch`. You can refer to [Launch Colossal-AI](../basics/launch_colossalai.md)
+for other initialization methods. We use `ProcessGroupMesh` to create tensor parallelism group and data parallelism group.
+
+```python
+# Distributed Enviroment
+config = {}
+colossalai.launch(config=config, rank=rank, world_size=world_size,host="localhost", port=port, backend="nccl")
+
+# Parallism Group
+tp_size, zero_size = 2, 2
+use_zero = True if zero_size > 1 else False
+proc_mesh = ProcessGroupMesh(tp_size, zero_size)
+tp_group, dp_group = proc_mesh.get_group_along_axis(0), proc_mesh.get_group_along_axis(1)
+```
+
+### step 3. Initialize Module and Optimizer
+Build our model. We created an MLP using two Linear Layer.
+
+```python
+# Init Llama from huggingface
+configuration = LlamaConfig()
+model = LlamaModel(configuration)
+dataset = load_tokenized_dataset(dataset_paths=args.dataset, mode="train")
+dataloader = plugin.prepare_dataloader(dataset, batch_size=8)
+criterion = lambda x: x.mean()
+dist_optim = DistributedAdaFactor(model.parameters())
+
+```
+
+### step 4.Init Booster
+
+```python
+plugin = LowLevelZeroPlugin()
+booster = Booster(plugin=plugin)
+model, dist_optim, criterion, dataloader, _ = booster.boost(model, dist_optim, criterion, dataloader)
+```
+### step 5.Train Your Model
+```python
+for epoch in range(max_epochs):
+    for input_ids, attention_mask in dataloader:
+        outputs = model(input_ids.cuda(), attention_mask.cuda())
+        loss = criterion(outputs.logits, input_ids)
+        booster.backward(loss, optimizer)
+        optimizer.step()
+        lr_scheduler.step()
+        optimizer.zero_grad()
+```
+
+## Supporting Information
+Model/Feature Compatibility Matrix:
```

Comment:
Please take plugin into consideration as well.

Reply:
> Please take plugin into consideration as well.


Added Plugin support info in https://github.com/hpcaitech/ColossalAI/pull/5484/commits/1038b233632c5ad04c40262a7027ba7205a14e65;